Makefile

```diff
@@ -2,12 +2,36 @@ VERSION = 0.1.0
 PREFIX = nginx-prometheus-exporter
 TAG = $(VERSION)
 GIT_COMMIT = $(shell git rev-parse --short HEAD)
-
+BUILD_DIR = build
 test:
 	go test ./...
 
+nginx-prometheus-exporter: test
+	CGO_ENABLED=0 go build -installsuffix cgo -ldflags "-X main.version=${VERSION} -X main.gitCommit=${GIT_COMMIT}" -o nginx-prometheus-exporter .
+
 container:
 	docker build --build-arg VERSION=$(VERSION) --build-arg GIT_COMMIT=$(GIT_COMMIT) -t $(PREFIX):$(TAG) . 
 
 push: container
-	docker push $(PREFIX):$(TAG)
\ No newline at end of file
+	docker push $(PREFIX):$(TAG)
+
+$(BUILD_DIR)/nginx-prometheus-exporter-linux-amd64:
+	GOARCH=amd64 CGO_ENABLED=0 GOOS=linux go build -installsuffix cgo -ldflags "-X main.version=${VERSION} -X main.gitCommit=${GIT_COMMIT}" -o $(BUILD_DIR)/nginx-prometheus-exporter-linux-amd64
+
+$(BUILD_DIR)/nginx-prometheus-exporter-linux-i386:
+	GOARCH=386 CGO_ENABLED=0 GOOS=linux go build -installsuffix cgo -ldflags "-X main.version=${VERSION} -X main.gitCommit=${GIT_COMMIT}" -o $(BUILD_DIR)/nginx-prometheus-exporter-linux-i386
+
+release: $(BUILD_DIR)/nginx-prometheus-exporter-linux-amd64 $(BUILD_DIR)/nginx-prometheus-exporter-linux-i386
+	mv $(BUILD_DIR)/nginx-prometheus-exporter-linux-amd64 $(BUILD_DIR)/nginx-prometheus-exporter && \
+	tar czf $(BUILD_DIR)/nginx-prometheus-exporter-${TAG}-linux-amd64.tar.gz -C $(BUILD_DIR) nginx-prometheus-exporter && \
+	rm $(BUILD_DIR)/nginx-prometheus-exporter
+	mv $(BUILD_DIR)/nginx-prometheus-exporter-linux-i386 $(BUILD_DIR)/nginx-prometheus-exporter && \
```

Comment:
please add en empty line between rm and mv to improve readability

Reply:
done